sdk/identity/identity/test/internal/node/usernamePasswordCredential.spec.ts

```diff
@@ -72,17 +65,12 @@ describe("UsernamePasswordCredential (internal)", function() {
   // To test this, please install @azure/msal-node-extensions and un-skip these tests.
   describe("Persistent tests", function() {
     try {
-      /* eslint-disable-next-line @typescript-eslint/no-require-imports */
-      require("@azure/msal-node-extensions");
+      requireMsalNodeExtensions();
     } catch (e) {
       return;
     }
 
     it("Persistence throws on Node 8, as expected", async function(this: Context) {
-      if (!isNode8) {
-        this.skip();
-      }
```

Comment:
Now that we have removed this, how do we ensure that this test is not run in other node versions?

Reply:
And since requireMsalNodeExtensions is called early in the describe block before any of the individual tests are run, and because all errors from requireMsalNodeExtensions is ignored, these tests will not be run at all. Ok.

Reply:
These tests can only run if we're on Node 10 and above and if we manually installed msal-node-extensions